test/ese/src/blue/src/esetest/esetest/bounce/loadcommon.cxx

```diff
@@ -0,0 +1,5082 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
This file starts with this:
    // Generated by generate_ese_stubs.pl
    #include "ese_common.hxx"

... in house. So I think you should keep that comment ... and maybe see if generate_ese_stubs.pl and still run and update this file.

Reply:
Do note you dropped the "Generated by" comment ...

Reply:
One more thing, probably other files have this as well

Reply:
The "generated" comment is also missing in other generated files, for now.
I tried running the script - it seems it hasn't been run in a while, since it results in a lot of whitespace differences. I'll update the script, though, but only with the header